pyensembl/ensembl_release.py

```diff
@@ -495,12 +505,13 @@ def transcript_name_of_transcript_id(self, transcript_id):
     #
     ###################################################
 
-    def _query_transcript_ids(self, property_name, value):
+    def _query_transcript_ids(self, property_name, value,
+                              feature='transcript'):
         results = self.db.query(
             select_column_names=['transcript_id'],
             filter_column=property_name,
             filter_value=value,
-            feature='transcript',
+            feature=feature,
```

Comment:
This is probably a needless generalization, since we're looking up distinct transcript IDs and I can't think of any feature we would to find those on other than transcripts. I guess this way you can ask "find the transcript ID of any exon which matches my filter criteria..."